sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -7,18 +7,20 @@
 import com.azure.core.amqp.AmqpTransportType;
 import com.azure.core.amqp.ProxyAuthenticationType;
 import com.azure.core.amqp.ProxyOptions;
-import com.azure.core.amqp.implementation.AzureTokenManagerProvider;
-import com.azure.core.amqp.implementation.CbsAuthorizationType;
-import com.azure.core.amqp.implementation.ConnectionOptions;
-import com.azure.core.amqp.implementation.ConnectionStringProperties;
 import com.azure.core.amqp.implementation.MessageSerializer;
-import com.azure.core.amqp.implementation.ReactorHandlerProvider;
-import com.azure.core.amqp.implementation.ReactorProvider;
+import com.azure.core.amqp.implementation.TracerProvider;
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
```

Comment:
This should be reordered. IntelliJ has a keyboard shortcut you can use too:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#optimize-imports

Reply:
According to your comment in the new revision amendments have been made.